.circleci/config.yml

```diff
@@ -27,7 +26,7 @@ jobs:
     <<: *defaults
     steps:
       - attach_workspace:
-          at: /go/src/github.com/gohugoio
```

Comment:
This now does not match the working_directory setting above.

Reply:
Did too many branches last night and cherry picked the wrong thing. The PR is updated to remove the `working_directory` key from the top.